roles/uperf-bench/tasks/main.yml

```diff
@@ -35,8 +35,18 @@
       - name = benchmark-operator
   register: bo
 
+- name: Capture ServiceIP
+  k8s_facts:
+    kind: Service
+    api_version: v1
+    namespace: '{{ operator_namespace }}'
+    label_selectors:
+      - type = uperf-bench-server-{{ trunc_uuid }}
+  register: serviceip
+  when: uperf.serviceip
+
```

Comment:
Wont this fail if it runs before the uperf-bench-server is created like on the first iteration through?

Reply:
The server doesn't need anything with the serviceip creation... I don't think it should matter (looking through the code)..

Reply:
Right. The question is really if k8s_facts returns nothing or an error if it doesnt find anything. I would assume nothing but  would want to be sure.